src/Samples/Sample.AspNetCore/Controllers/PaymentController.cs

```diff
@@ -238,8 +261,15 @@ public async Task<IActionResult> Reversal(string paymentId, PaymentInstrument in
                             Amount.FromDecimal(0), description, DateTime.Now.Ticks.ToString());
                         response = await cardPayment.Operations.Reverse.Invoke(cardReversal);
                         break;
+                    case PaymentInstrument.Trustly:
+                        var trustlyPayment = await this.swedbankPayClient.Payments.TrustlyPayments.Get(new Uri(paymentId, UriKind.RelativeOrAbsolute));
+                        var trustlyReversal = new SwedbankPay.Sdk.Payments.TrustlyPayments.TrustlyPaymentReversalRequest(
+                            Operation.Sale, Amount.FromDecimal(order.Lines.Sum(e => e.Quantity * e.Product.Price)),
+                            Amount.FromDecimal(0), DateTime.Now.Ticks.ToString(), "receipt reference", description);
+                        response = await trustlyPayment.Operations.Reverse.Invoke(trustlyReversal);
```

Comment:
If we move the other out this could also follow 😃 
This also applies to the cardPayment part of course 😄